### PR TITLE
[#20] Make sure /fileinfo and /files/ handles all URLs specified in the PDF

### PR DESCRIPTION
**Pull request by tessi**
Collecting all input URLS/identifiers from our specification PDF in a huge integration test to compare our fileData service against known inputs and see how it behaves.

### CODE REVIEW

**Comment by tessi** [comment]
This is work in progress. Current state of the tests is:

```
FileData
  for an input URL which normalizes to https://commons.wikimedia.org/
    ✅ requesting info for File:Helene_Fischer_2010.jpg (264ms)
    🔥 requesting info for https://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg (8ms)
    🔥 requesting info for https://commons.m.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg (1ms)
    🔥 requesting info for http://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg
    🔥 requesting info for commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg
    🔥 requesting info for https://commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg (1ms)
    🔥 requesting info for https://commons.m.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg
    🔥 requesting info for http://commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg
    🔥 requesting info for commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg
    ✅ requesting info for https://upload.wikimedia.org/wikipedia/commons/8/84/Helene_Fischer_2010.jpg (192ms)
    ✅ requesting info for http://upload.wikimedia.org/wikipedia/commons/8/84/Helene_Fischer_2010.jpg (183ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/commons/8/84/Helene_Fischer_2010.jpg
    ✅ requesting info for https://upload.wikimedia.org/wikipedia/commons/thumb/8/84/Helene_Fischer_2010.jpg/171px-Helene_Fischer_2010.jpg (194ms)
    ✅ requesting info for http://upload.wikimedia.org/wikipedia/commons/thumb/8/84/Helene_Fischer_2010.jpg/171px-Helene_Fischer_2010.jpg (211ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/commons/thumb/8/84/Helene_Fischer_2010.jpg/171px-Helene_Fischer_2010.jpg
    🔥 requesting info for https://commons.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg
    🔥 requesting info for https://commons.m.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg
    🔥 requesting info for http://commons.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg
    🔥 requesting info for commons.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg
    🔥 requesting info for https://commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg&uselang=de (1ms)
    🔥 requesting info for https://commons.m.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg&uselang=de
    🔥 requesting info for commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg&uselang=de
    🔥 requesting info for https://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar
    🔥 requesting info for https://commons.m.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar (1ms)
    🔥 requesting info for http://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar
    🔥 requesting info for commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar
  for an input URL which normalizes to https://de.wikipedia.org/
    🔥 requesting info for https://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (5007ms)
    🔥 requesting info for https://de.m.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (218ms)
    🔥 requesting info for http://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (215ms)
    🔥 requesting info for de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg
    🔥 requesting info for https://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (190ms)
    🔥 requesting info for https://de.m.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (197ms)
    🔥 requesting info for http://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (186ms)
    🔥 requesting info for de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg
    🔥 requesting info for https://upload.wikimedia.org/wikipedia/de/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (188ms)
    🔥 requesting info for http://upload.wikimedia.org/wikipedia/de/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (186ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/de/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg
    🔥 requesting info for https://upload.wikimedia.org/wikipedia/de/thumb/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg/320px-1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (200ms)
    🔥 requesting info for http://upload.wikimedia.org/wikipedia/de/thumb/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg/320px-1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (198ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/de/thumb/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg/320px-1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (1ms)
    🔥 requesting info for https://de.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (187ms)
    🔥 requesting info for https://de.m.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (199ms)
    🔥 requesting info for http://de.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (205ms)
    🔥 requesting info for de.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg
    🔥 requesting info for https://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (204ms)
    🔥 requesting info for https://de.m.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (190ms)
    🔥 requesting info for http://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (198ms)
    🔥 requesting info for de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (1ms)
    🔥 requesting info for https://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (192ms)
    🔥 requesting info for https://de.m.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (200ms)
    🔥 requesting info for http://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (190ms)
    🔥 requesting info for de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (1ms)
```

**Comment by tessi** [comment]
Added the normalized title (unescaped umlauts, replaces underscores etc.) to the fileData output. We're using this as the new input for the attribution now. In addition, this gives us more positive examples in our big list of integration tests :)

current state:

```
FileData
  for an input URL which normalizes to https://commons.wikimedia.org/
    ✅ requesting info for File:Helene_Fischer_2010.jpg (249ms)
    🔥 requesting info for https://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg (12ms)
    🔥 requesting info for https://commons.m.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg
    🔥 requesting info for http://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg (1ms)
    🔥 requesting info for commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg
    🔥 requesting info for https://commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg (1ms)
    🔥 requesting info for https://commons.m.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg
    🔥 requesting info for http://commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg
    🔥 requesting info for commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg
    ✅ requesting info for https://upload.wikimedia.org/wikipedia/commons/8/84/Helene_Fischer_2010.jpg (189ms)
    ✅ requesting info for http://upload.wikimedia.org/wikipedia/commons/8/84/Helene_Fischer_2010.jpg (195ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/commons/8/84/Helene_Fischer_2010.jpg
    ✅ requesting info for https://upload.wikimedia.org/wikipedia/commons/thumb/8/84/Helene_Fischer_2010.jpg/171px-Helene_Fischer_2010.jpg (191ms)
    ✅ requesting info for http://upload.wikimedia.org/wikipedia/commons/thumb/8/84/Helene_Fischer_2010.jpg/171px-Helene_Fischer_2010.jpg (190ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/commons/thumb/8/84/Helene_Fischer_2010.jpg/171px-Helene_Fischer_2010.jpg (1ms)
    🔥 requesting info for https://commons.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg
    🔥 requesting info for https://commons.m.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg
    🔥 requesting info for http://commons.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg
    🔥 requesting info for commons.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg
    🔥 requesting info for https://commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg&uselang=de (1ms)
    🔥 requesting info for https://commons.m.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg&uselang=de
    🔥 requesting info for commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg&uselang=de
    🔥 requesting info for https://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar (1ms)
    🔥 requesting info for https://commons.m.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar
    🔥 requesting info for http://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar (1ms)
    🔥 requesting info for commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar
  for an input URL which normalizes to https://de.wikipedia.org/
    ✅ requesting info for https://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (216ms)
    ✅ requesting info for https://de.m.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (190ms)
    ✅ requesting info for http://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (202ms)
    🔥 requesting info for de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg
    ✅ requesting info for https://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (201ms)
    ✅ requesting info for https://de.m.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (187ms)
    ✅ requesting info for http://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (188ms)
    🔥 requesting info for de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg
    ✅ requesting info for https://upload.wikimedia.org/wikipedia/de/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (195ms)
    ✅ requesting info for http://upload.wikimedia.org/wikipedia/de/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (202ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/de/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (1ms)
    ✅ requesting info for https://upload.wikimedia.org/wikipedia/de/thumb/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg/320px-1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (188ms)
    ✅ requesting info for http://upload.wikimedia.org/wikipedia/de/thumb/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg/320px-1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (197ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/de/thumb/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg/320px-1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (1ms)
    ✅ requesting info for https://de.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (188ms)
    ✅ requesting info for https://de.m.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (189ms)
    ✅ requesting info for http://de.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (199ms)
    🔥 requesting info for de.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg
    ✅ requesting info for https://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (187ms)
    ✅ requesting info for https://de.m.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (191ms)
    ✅ requesting info for http://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (201ms)
    🔥 requesting info for de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (1ms)
    ✅ requesting info for https://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (199ms)
    ✅ requesting info for https://de.m.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (194ms)
    ✅ requesting info for http://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (194ms)
    🔥 requesting info for de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (1ms)
```

28 tests to go

**Comment by tessi** [comment]
Added basic commons-urls support. We're now down to 19 failing tests:

```
FileData
  for an input URL which normalizes to https://commons.wikimedia.org/
    ✅ requesting info for File:Helene_Fischer_2010.jpg (269ms)
    ✅ requesting info for https://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg (201ms)
    ✅ requesting info for https://commons.m.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg (189ms)
    ✅ requesting info for http://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg (412ms)
    🔥 requesting info for commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg (11ms)
    🔥 requesting info for https://commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg (3ms)
    🔥 requesting info for https://commons.m.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg (1ms)
    🔥 requesting info for http://commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg
    🔥 requesting info for commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg
    ✅ requesting info for https://upload.wikimedia.org/wikipedia/commons/8/84/Helene_Fischer_2010.jpg (225ms)
    ✅ requesting info for http://upload.wikimedia.org/wikipedia/commons/8/84/Helene_Fischer_2010.jpg (189ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/commons/8/84/Helene_Fischer_2010.jpg (1ms)
    ✅ requesting info for https://upload.wikimedia.org/wikipedia/commons/thumb/8/84/Helene_Fischer_2010.jpg/171px-Helene_Fischer_2010.jpg (186ms)
    ✅ requesting info for http://upload.wikimedia.org/wikipedia/commons/thumb/8/84/Helene_Fischer_2010.jpg/171px-Helene_Fischer_2010.jpg (286ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/commons/thumb/8/84/Helene_Fischer_2010.jpg/171px-Helene_Fischer_2010.jpg (1ms)
    ✅ requesting info for https://commons.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg (194ms)
    ✅ requesting info for https://commons.m.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg (212ms)
    ✅ requesting info for http://commons.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg (190ms)
    🔥 requesting info for commons.wikimedia.org/wiki/Helene_Fischer#/media/File:Helene_Fischer_2010.jpg
    🔥 requesting info for https://commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg&uselang=de (1ms)
    🔥 requesting info for https://commons.m.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg&uselang=de
    🔥 requesting info for commons.wikimedia.org/w/index.php?title=File:Helene_Fischer_2010.jpg&uselang=de (1ms)
    ✅ requesting info for https://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar (191ms)
    ✅ requesting info for https://commons.m.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar (399ms)
    ✅ requesting info for http://commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar (253ms)
    🔥 requesting info for commons.wikimedia.org/wiki/File:Helene_Fischer_2010.jpg?foo=bar (1ms)
  for an input URL which normalizes to https://de.wikipedia.org/
    ✅ requesting info for https://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (238ms)
    ✅ requesting info for https://de.m.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (410ms)
    ✅ requesting info for http://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (186ms)
    🔥 requesting info for de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg
    ✅ requesting info for https://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (212ms)
    ✅ requesting info for https://de.m.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (192ms)
    ✅ requesting info for http://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (189ms)
    🔥 requesting info for de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (1ms)
    ✅ requesting info for https://upload.wikimedia.org/wikipedia/de/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (268ms)
    ✅ requesting info for http://upload.wikimedia.org/wikipedia/de/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (199ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/de/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (1ms)
    ✅ requesting info for https://upload.wikimedia.org/wikipedia/de/thumb/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg/320px-1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (196ms)
    ✅ requesting info for http://upload.wikimedia.org/wikipedia/de/thumb/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg/320px-1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (197ms)
    🔥 requesting info for upload.wikimedia.org/wikipedia/de/thumb/f/fb/1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg/320px-1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (1ms)
    ✅ requesting info for https://de.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (209ms)
    ✅ requesting info for https://de.m.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (422ms)
    ✅ requesting info for http://de.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg (217ms)
    🔥 requesting info for de.wikipedia.org/wiki/S%C3%BCddeutsche_Fu%C3%9Fballmeisterschaft_1901/02#/media/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg
    ✅ requesting info for https://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (192ms)
    ✅ requesting info for https://de.m.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (192ms)
    ✅ requesting info for http://de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (409ms)
    🔥 requesting info for de.wikipedia.org/w/index.php?title=File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg&uselang=de (1ms)
    ✅ requesting info for https://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (252ms)
    ✅ requesting info for https://de.m.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (182ms)
    ✅ requesting info for http://de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (191ms)
    🔥 requesting info for de.wikipedia.org/wiki/File:1_FC_Bamberg_-_1_FC_N%C3%BCrnberg_1901.jpg?foo=bar (1ms)


```

File: models/attribution.test.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ describe('attribution', () => {
     fileInfo: {
       rawUrl: 'https://commons.wikimedia.org/wiki/File:Eisklettern_kl_engstligenfall.jpg',
       title: 'File:Eisklettern kl engstligenfall.jpg',
```
Comment (juffel) [inline models/attribution.test.js]: Is the `title` still used somewhere or can it be removed completetely?

Comment (juffel) [reply models/attribution.test.js]: Nevermind, looks like `routes/fileinfo.js` uses the `title` attribute for requesting the license.

Comment (juffel) [reply models/attribution.test.js]: Looks like it can be removed to me.

**Pull request merged**